profile_library/eglo/FUEVA-Z/model.json

```
@@ -0,0 +1,19 @@
+{
+  "aliases": [
+    "33957",
+    "AwoX LED light with color temperature"
```

```suggestion
    "33957"
```

This alias can be removed.
Powercalc will also match model id in parenthesis.
As seen in your device info the complete model is exposed as `LED light with color temperature (33957)`.